src/components/dom.js

```diff
@@ -104,46 +158,60 @@ define([], function () {
 
             if (!windowSizeEventsBound) {
                 windowSizeEventsBound = true;
-                addEventListenerWithOptions(window, "orientationchange", clearWindowSize, { passive: true });
-                addEventListenerWithOptions(window, 'resize', clearWindowSize, { passive: true });
+                addEventListener(window, "orientationchange", clearWindowSize, { passive: true });
+                addEventListener(window, 'resize', clearWindowSize, { passive: true });
             }
         }
 
         return windowSize;
     }
 
-    var standardWidths = [480, 720, 1280, 1440, 1920, 2560, 3840, 5120, 7680];
-    function getScreenWidth() {
-        var width = window.innerWidth;
-        var height = window.innerHeight;
+    /**
+     * Standard screen widths.
+     */
+    const standardWidths = [480, 720, 1280, 1440, 1920, 2560, 3840, 5120, 7680];
+
+    /**
+     * Returns screen width.
+     * @returns {number} Screen width
+     */
+    export function getScreenWidth() {
+        let width = window.innerWidth;
+        const height = window.innerHeight;
 
         if (height > width) {
             width = height * (16.0 / 9.0);
         }
 
-        var closest = standardWidths.sort(function (a, b) {
+        const closest = standardWidths.sort(function (a, b) {
             return Math.abs(width - a) - Math.abs(width - b);
         })[0];
 
         return closest;
     }
 
-    var _animationEvent;
-    function whichAnimationEvent() {
+    /**
+     * Name of animation end event.
+     */
+    let _animationEvent;
 
+    /**
+     * Returns name of animation end event.
+     * @returns {string} Name of animation end event
```

Comment:
@MrTimscampi period or no period?

Reply:
Currently it looks like this
![doc1](https://user-images.githubusercontent.com/56478732/77999570-8e8b9b00-733b-11ea-9c91-ac1af0bb8d57.png)

Reply:
I'd add a period (that's what I've been doing on every JSDoc comment).

I feel like we should be writing these both with in-source legibility (hence my use of "-" to separate parameter names and descriptions) and periods (to look nicer in the compiled docs).

Eventually, we probably want the JSDoc to be available in the developer doc to help people understand the web client before diving in.

Reply:
![doc2](https://user-images.githubusercontent.com/56478732/78010032-91da5300-734a-11ea-8e4d-597ee7fd306b.png)

Reply:
As discussed on Matrix with @dkanada, adopting the Google Style Guide for JSDoc is probably a good idea: https://google.github.io/styleguide/jsguide.html#jsdoc

We're nearly there anyway, so...

We'll make an addition to the developer documentation about it at some point.

Reply:
Yeah, we can ignore this change for now and then fix all issues with the linting.